docs/getting-started/create-comit-app.md

```diff
@@ -4,6 +4,286 @@ title: Getting Started
+success Saved lockfile.
+✨  Done in 92.81s.
+```
+
+Let's check out the provided `package.json` for any useful scripts:
+
+```
+▶ cat package.json
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "start": "node ./src/index.js",
+    "start-env": "comit-scripts start-env",
+    "force-clean-env": "comit-scripts force-clean-env"
+  },
+  "dependencies": {
+    "comit-sdk": "^0.10.0"
+  },
+  "devDependencies": {
+    "comit-scripts": "^0.8.3"
+  }
+}
+```
+
+As you can see, two scripts are using `comit-scripts`, let's run `start-env`:
+
+```
+▶ yarn run start-env
+yarn run v1.21.1
+$ comit-scripts start-env
+Creating Docker network (create-comit-app)...✓
+Starting Ethereum node...✓
+Starting Bitcoin node...✓
+Starting two cnds...✓
+Writing configuration in env file...✓
+🎉 Environment is ready, time to create a COMIT app!
+✓
+```
+
+So, what does it do?
+
+`Creating Docker network (create-comit-app)...✓`: creates a docker network to allow several docker containers to communicate with each others. The network is named `create-comit-app`.
+
+
+`Starting Ethereum node...✓`:
+- Starts a parity node in a docker container,
+- Deploys an ERC20 token contract
+- Creates two accounts,
+- Funds each account with 1000 Ethers and 100 ERC20 tokens.
+
+`Starting Bitcoin node...✓`"
+- Starts a bitcoind node in a docker container,
+- Create two HD wallet,
+- Funds each wallet with 10 Bitcoin,
+- Run `generate` every second to mine a new block.
+
+`Starting two cnds...✓`: Starts 2 cnd (COMIT Network Daemon) in docker containers.
+
+`Writing configuration in env file...✓`: Write useful information to `~/.create-comit-app/env`
+
+Let's check out the result:
+
+```
+▶ docker ps
+CONTAINER ID        IMAGE                             COMMAND                  CREATED             STATUS              PORTS                                                                                                  NAMES
+207bdd00d103        comitnetwork/cnd:0.4.0            "tini -- cnd --confi…"   26 minutes ago      Up 26 minutes       8000/tcp, 9939/tcp, 0.0.0.0:58249->8080/tcp                                                            cnd_1
+e72b5910e530        comitnetwork/cnd:0.4.0            "tini -- cnd --confi…"   26 minutes ago      Up 26 minutes       8000/tcp, 9939/tcp, 0.0.0.0:58248->8080/tcp                                                            cnd_0
+7c9855e6f3c0        coblox/bitcoin-core:0.17.0        "/usr/bin/bitcoind -…"   26 minutes ago      Up 26 minutes       8332-8333/tcp, 18332-18333/tcp, 0.0.0.0:58234->18443/tcp, 0.0.0.0:58233->18444/tcp                     bitcoin
+66a52568666d        coblox/parity-poa:v2.5.9-stable   "/bin/parity"            26 minutes ago      Up 26 minutes       5001/tcp, 8080/tcp, 8082-8083/tcp, 8180/tcp, 8546/tcp, 30303/tcp, 30303/udp, 0.0.0.0:58176->8545/tcp   ethereum
+```
+Indeed we have bitcoind, parity and cnds running.
+
+Let's have a look at this `env` file:
```

Comment:
This block should be moved up to line 190 after 
```
`Writing configuration in env file...✓`: Write useful information to `~/.create-comit-app/env`
```

Reply:
I am not sure this would make sense. Where would you put the `docker ps` section then?

Reply:
Proposed new solution: Move the docker ps section to the end of `So, what does it do?`, before this sentence:
```
To stop the environment, just hit `CTRL+C` and wait few seconds:
```

(For me it does not really fit here, as this part concerns the `env` file.)

Reply:
I made this section more structured, please have a look, it should resolve your conncers.